musicAlbum/musicalbum.rb

```diff
@@ -0,0 +1,12 @@
+require_relative '../item'
+
+class MusicAlbum
+  def initialize(_on_spotify)
```

Comment:
- [ ] As shown in the UML class diagram `on_spotiy` is a public instance variable. Please add getter and setter method for it. You can use `attr_accessor` for it.

Reply:
Thank you for pointing this one out. I have resolved it.